### PR TITLE
dlgBoxplot

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by stevekogo** [comment]
@maxwellfundi What is the current state of this?

**Comment by maxwellfundi** [comment]
@africanmathsinitiative/developers this is ready for reviews. 

Thanks.

**Comment by lilyclements** [comment]
Ive made some changes to merge, but it wasn't thorough enough. This needs a much more thorough review.
Also, I think the main receiver has a bug, you're telling it to set numeric but it isn't setting numeric only. We saw this problem on Histogram too where it causes a crash.

**Comment by maxwellfundi** [comment]
@lilyclements yes, i have realized that this has that the main receiver has that bug. Let me investigate it actually.

File: instat/dlgBoxPlot.vb [hunk 1]
```diff
@@ -13,137 +13,178 @@
...
+        ucrSecondFactorReceiver.SetParameterIsString()
+        ucrSecondFactorReceiver.bWithQuotes = False
+
+        ucrChkVarWidth.SetParameter(New RParameter("varwidth", 1))
+        ucrChkVarWidth.SetText("Variable Width")
+        ucrChkVarWidth.SetValuesCheckedAndUnchecked("TRUE", "FALSE")
+        ucrChkVarWidth.SetRDefault("FALSE")
+
+        clsCoordFlipFunc.SetPackageName("ggplot2")
+        clsCoordFlipFunc.SetRCommand("coord_flip")
```
Comment (lilyclements) [inline instat/dlgBoxPlot.vb]: Just wondering, what is the benefit to setting this in `Initialise `as opposed to `SetDefaults`?

Comment (maxwellfundi) [inline instat/dlgBoxPlot.vb]: We are defining a paremeter for the ucrChkHorizontalBoxplot  which is a  function. Which we are setting it below on line 95. We can set a parameter that does not exist and so we have to do it here .

Comment (lilyclements) [inline instat/dlgBoxPlot.vb]: Ah okay, I see! Thank you

**Comment by lilyclements** [comment]
@dannyparsons This is ready for review

**Pull request merged**